src/components/PreviewColumn.jsx

```diff
@@ -0,0 +1,92 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+
+import ArticlePreview from './ArticlePreview';
+import LinkWithArrow from './LinkWithArrow';
+
+import {
+  previewColumn,
+  columnItems,
+  linkArrow,
+} from '../styles/previewColumn.module.scss';
+
+const PreviewColumn = ({
+  columnPreviewData,
+  previewImageEdges,
+  authorImageEdges,
+  header,
+  className,
+}) => {
+  const renderColumn = () => (
+    <ul className={className ? `${columnItems} ${className}` : columnItems}>
+      {columnPreviewData.map(
+        ({
+          title,
+          articleUrl,
+          tags,
+          date,
+          readingTime,
+          previewImageName,
+          authorName,
+          authorCredentials,
+          authorUrl,
+          authorImageName,
+          description,
+        }) => {
+          const previewImage = previewImageEdges.find(
+            (edge) => edge.node.base === previewImageName
+          );
+          const authorImage = authorImageEdges.find(
+            (edge) => edge.node.base === authorImageName
+          );
+          return (
+            <li key={title}>
+              <ArticlePreview
+                title={title}
+                tags={tags}
+                articleUrl={articleUrl}
+                date={date}
+                description={description}
+                readingTime={readingTime}
+                previewImage={previewImage}
+                authorName={authorName}
+                authorCredentials={authorCredentials}
+                authorUrl={authorUrl}
+                authorImage={authorImage}
+              />
+            </li>
+          );
+        }
+      )}
+    </ul>
+  );
+
+  return (
+    <div className={previewColumn}>
+      {header && <h2>{header.toUpperCase()}</h2>}
```

Comment:
css property `text-transform: uppercase` is better for both code readability and screen readers